R/forestplot.R

```diff
@@ -11,19 +11,16 @@
 #' @export
 #'
 #' @examples
-#' mae <- hermes::multi_assay_experiment
-#' adtte <- teal.modules.hermes::rADTTE %>%
-#'   dplyr::mutate(is_event = (.data$CNSR == 0))
+#' data <- teal_data()
+#' data <- within(data, {
+#'   ADTTE <- teal.modules.hermes::rADTTE %>%
+#'     dplyr::mutate(is_event = .data$CNSR == 0)
+#'   MAE <- hermes::multi_assay_experiment
+#' })
+#' datanames <- c("ADTTE", "MAE")
+#' datanames(data) <- datanames
+#' data@join_keys <- cdisc_join_keys(!!!datanames)
```

Comment:
this might be removed, once joinkeys PR is finalized https://github.com/insightsengineering/teal.data/pull/184

Reply:
changed in https://github.com/insightsengineering/teal.modules.hermes/pull/342/commits/ba3eb957c3903928f19d1ae6b5ab09c162ebc866